libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -65,7 +65,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
     // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:Hard,2:Soft
+    // @RebootRequired: True
     // @User: Advanced
     AP_GROUPINFO("AUTO_SWITCH", 3, AP_GPS, _auto_switch, 1),
```

Comment:
I think Tridge and others will be happier if we leave the existing "UseBest" logic (that's what I'm calling it) for a while until the "Blending" has had more testing by use and beta testers.